### ISSUE TITLE
TODO in `src/igraph_vector.hpp`

### ISSUE THREAD

**Issue by maelle** [igraph/rigraph#1459]
@Antonov548 what to do about https://github.com/igraph/rigraph/blob/15a70e6b6fe5ec7834f81307ed97bcc634bd66a8/src/igraph_vector.hpp#L44?

**Comment by szhorvat** [comment]
This is a longer-term thing. We will use the new C++ wrapper which is being developed at https://github.com/igraph/igraph-cpp but isn't ready for use yet.

**Comment by maelle** [comment]
Should this comment be removed as we won't forget to do that then? Or am I misunderstanding?

**Comment by Antonov548** [comment]
> Should this comment be removed as we won't forget to do that then? Or am I misunderstanding?

Yes, it was just prototype for C++ wrapper. I guess this comment could be removed.

**Comment by maelle** [comment]
@szhorvat would you vote for keeping or removing it? 

Thanks both!
